bin/rake

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby.exe
+require_relative '../config/boot'
+require 'rake'
+Rake.application.run

```

Comment:
- You have done great so far :+1: But there are still some requirements you need to meet before moving to the next milestone. :+1: :100: 
- [ ] Just as the previous reviewer pointed out, I still can't start the application, After `Creating, Migrating and seeding the database` I ran the cmd `./bin/dev`, I got the error shown in the screenshot below. I noticed that the `dev` file is missing in the `bin` folder. also, you are missing the `Procfile.dev` file on the root of your project. I'll suggest you refer to the project setup and article one more time to get your app in working condition. :+1: :smile: :wave: 

- - Screenshot reference:
![Screenshot from 2023-11-02 16-37-23](https://github.com/rodrigopenayo98/hello-rails-react/assets/107632780/d6e00f65-eba3-484b-ad65-a1c50ce4bb23)


## I noticed you closed the initial PR #2  and created a new one. This is not the right process. You are not meant to close a PR until it have been approved and merged. :+1: